plugins/block-plus-minus/src/if.js

```diff
@@ -123,22 +123,22 @@ const controlsIfMutator = {
    * Appears to remove the input at the given index. Actually shifts attached
    * blocks and then removes the input at the bottom of the block. This is to
    * make sure the inputs are always IF0, IF1, etc with no gaps.
-   * @param {number?} opt_index The index of the input to "remove", or undefined
+   * @param {number?} index The index of the input to "remove", or undefined
```

Comment:
According to google style guide ? Should be before the type like
?number and not number? , Should i correct this?

Reply:
Yes, that would be great. Thanks!

Reply:
I already did that

Reply:
@alschmiedt So, this should be `?number=` right? , Because it is also optional

Reply:
Yes. Good catch!